docs/products/kafka/howto/kcat.rst

```diff
@@ -1,34 +1,35 @@
-Use kcat with Aiven for Apache Kafka®
-=====================================
+Use `kcat` with Aiven for Apache Kafka®
```

Comment:
```suggestion
Use ``kcat`` with Aiven for Apache Kafka®
```

Reply:
Well spotted. Also fixed elsewhere in the same file.